src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -398,7 +400,16 @@ public String getQuestionResultStatisticsHtml(
 
         DecimalFormat df = new DecimalFormat("#.##");
         
-        for (Entry<String, List<Integer>> entry : optionPoints.entrySet()) {
+        SortedMap<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>(new Comparator<String>() {
+            @Override
+            public int compare(String s1, String s2) {
+                return s2.compareTo(s1);
+            }
+        });
+        
+        sortedOptionPoints.putAll(optionPoints);
```

Comment:
Perhaps the sorting (lines 403-410) could be performed in the `generateOptionPointsMapping` method so that both the html and csv statistics will be sorted :)